PushSharp.Apple/Exceptions.cs

```diff
@@ -79,6 +85,11 @@ public override string ToString()
 
 			return string.Format("APNS NotificationFailureException -> {0} : {1} -> {2}", ErrorStatusCode, ErrorStatusDescription, nstr);
 		}
+
+		public override string ToString()
+		{
+			return string.Format("APNS NotificationFailureException -> {0} : {1}", ErrorStatusCode, ErrorStatusDescription);
+		}
```

Comment:
Just confirming that you _do_ want to drop the serialized notification JSON from the message?

Oh, because it's already logged by the exception handler, so this is preventing unnecessary duplication.